contents/spotlight/startup-puzzle.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "How Puzzle defined its ICP and built a product around it"
+date: 2024-02-01
+author: ["joe-martin"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/startup_puzzle.jpg
+featuredImageType: full
+category: Startups
+---
+
+> Each month, we speak to a team enrolled in PostHog for Startups about the way they work. This time, we spoke to [Puzzle](https://puzzle.io/), an accountancy platform for early-stage startups, about how they define and use their Ideal Customer Persona (ICP). 
+
+Building an Ideal Customer Persona (sometimes known as an Ideal Buyer Profile) is one of the most challenging but essential steps for growing your product. Not only will it help you make product and pricing decisions, it will also clarify your value proposition and dictate how you judge successes and failures.
```

Comment:
We generally refer to ICP as ideal customer profile (ICP), not a persona, and a buyer profile is something different. Later on in the piece they talk about their ICP as a person, so kind of sounds like they have a buyer persona, not an ICP? 

In any case, I think it's important we're internally consistent about what ICP means to us, so maybe we frame this around the buyer persona bit and not the ICP bit?